nxdrive/manager.py

```diff
@@ -508,6 +510,24 @@ def set_direct_edit_auto_lock(self, value: bool) -> None:
         log.debug(f"Changed parameter 'direct_edit_auto_lock' to {value}")
         self.dao.store_bool("direct_edit_auto_lock", value)
 
+    @pyqtSlot(str, result=bool)
+    def get_feature_state(self, name: str) -> bool:
+        return Feature.__dict__[name]
+
+    @pyqtSlot(bool, str)
+    def set_feature_state(self, value: bool, name: str) -> None:
```

Comment:
Could you change the arguments order? The name first, then the value (to keep things homogene with other methods).